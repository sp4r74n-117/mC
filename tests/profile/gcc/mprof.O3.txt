function main
 called from ??
 took avg: 10416 stddev: 0 cycles
function sum
 called from ??
 took avg: 33 stddev: 8 cycles
