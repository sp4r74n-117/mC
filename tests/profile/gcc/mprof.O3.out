0x8048c07 0xf75c972e 52
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 96
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 24
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048c07 0xf75c972e 28
0x8048c07 0xf75c972e 32
0x8048c07 0xf75c972e 36
0x8048590 0xf75c972e 10416
